There are test failures
```
[ERROR] Failures: 
[ERROR]   MVELToDataWrapperTranslatorTest.testFulfillmentGroupQualificationDataWrapper:272
[ERROR]   MVELToDataWrapperTranslatorTest.testNestedExpressionForFulfillmentGroupQualificationDataWrapper:304
[ERROR] Errors: 
[ERROR]   DataDTOToMVELTranslatorTest.testFulfillmentQualificationMVEL:268 » MVELTranslation
[INFO] 
[ERROR] Tests run: 23, Failures: 2, Errors: 1, Skipped: 0
```

--- a/admin/broadleaf-admin-module/src/test/java/org/broadleafcommerce/admin/web/rulebuilder/DataDTOToMVELTranslatorTest.java
+++ b/admin/broadleaf-admin-module/src/test/java/org/broadleafcommerce/admin/web/rulebuilder/DataDTOToMVELTranslatorTest.java
@@ -253,7 +253,7 @@ public class DataDTOToMVELTranslatorTest extends TestCase {
         dataDTO.setCondition(BLCOperator.AND.name());
 
         ExpressionDTO e1 = new ExpressionDTO();
-        e1.setId("address.state.name");
+        e1.setId("address.stateProvinceRegion");
         e1.setOperator(BLCOperator.EQUALS.name());
         e1.setValue("Texas");
 
@@ -266,7 +266,7 @@ public class DataDTOToMVELTranslatorTest extends TestCase {
         dataDTO.getRules().add(e2);
 
         String translated = translator.createMVEL("fulfillmentGroup", dataDTO, fulfillmentGroupFieldService);
-        String mvel = "?fulfillmentGroup.?address.?state.?name==\"Texas\"&&(?fulfillmentGroup.?retailFulfillmentPrice.getAmount()>=99&&(?fulfillmentGroup.?retailFulfillmentPrice.getAmount()<=199))";
+        String mvel = "?fulfillmentGroup.?address.?stateProvinceRegion==\"Texas\"&&(?fulfillmentGroup.?retailFulfillmentPrice.getAmount()>=99&&(?fulfillmentGroup.?retailFulfillmentPrice.getAmount()<=199))";
         assert (mvel.equals(translated));
     }
 

--- a/admin/broadleaf-admin-module/src/test/java/org/broadleafcommerce/admin/web/rulebuilder/MVELToDataWrapperTranslatorTest.java
+++ b/admin/broadleaf-admin-module/src/test/java/org/broadleafcommerce/admin/web/rulebuilder/MVELToDataWrapperTranslatorTest.java
@@ -260,7 +260,7 @@ public class MVELToDataWrapperTranslatorTest extends TestCase {
         Property[] properties = new Property[1];
         Property mvelProperty = new Property();
         mvelProperty.setName("matchRule");
-        mvelProperty.setValue("fulfillmentGroup.address.state.name==\"Texas\"&&fulfillmentGroup.retailFulfillmentPrice.getAmount()>=99&&fulfillmentGroup.retailFulfillmentPrice.getAmount()<=199");
+        mvelProperty.setValue("fulfillmentGroup.address.stateProvinceRegion==\"Texas\"&&fulfillmentGroup.retailFulfillmentPrice.getAmount()>=99&&fulfillmentGroup.retailFulfillmentPrice.getAmount()<=199");
         properties[0] = mvelProperty;
         Entity[] entities = new Entity[1];
         Entity entity = new Entity();
@@ -276,7 +276,7 @@ public class MVELToDataWrapperTranslatorTest extends TestCase {
 
         assert(dataWrapper.getData().get(0).getRules().get(0) instanceof ExpressionDTO);
         ExpressionDTO e1 = (ExpressionDTO) dataWrapper.getData().get(0).getRules().get(0);
-        assert(e1.getId().equals("address.state.name"));
+        assert(e1.getId().equals("address.stateProvinceRegion"));
         assert(e1.getOperator().equals(BLCOperator.EQUALS.name()));
         assert(e1.getValue().equals("Texas"));
 
@@ -293,7 +293,7 @@ public class MVELToDataWrapperTranslatorTest extends TestCase {
         Property[] properties = new Property[1];
         Property mvelProperty = new Property();
         mvelProperty.setName("matchRule");
-        mvelProperty.setValue("fulfillmentGroup.address.state.name==\"Texas\"&&(fulfillmentGroup.retailFulfillmentPrice.getAmount()>=99&&fulfillmentGroup.retailFulfillmentPrice.getAmount()<=199)");
+        mvelProperty.setValue("fulfillmentGroup.address.stateProvinceRegion==\"Texas\"&&(fulfillmentGroup.retailFulfillmentPrice.getAmount()>=99&&fulfillmentGroup.retailFulfillmentPrice.getAmount()<=199)");
         properties[0] = mvelProperty;
         Entity[] entities = new Entity[1];
         Entity entity = new Entity();
@@ -308,7 +308,7 @@ public class MVELToDataWrapperTranslatorTest extends TestCase {
 
         assert(dataWrapper.getData().get(0).getRules().get(0) instanceof ExpressionDTO);
         ExpressionDTO e1 = (ExpressionDTO) dataWrapper.getData().get(0).getRules().get(0);
-        assert(e1.getId().equals("address.state.name"));
+        assert(e1.getId().equals("address.stateProvinceRegion"));
         assert(e1.getOperator().equals(BLCOperator.EQUALS.name()));
         assert(e1.getValue().equals("Texas"));
 

